src/components/general/DatePicker.js

```diff
@@ -15,99 +16,76 @@ class DatePicker extends Component {
 			month: date.length === 3 ? date[1] : "",
 			day: date.length === 3 ? date[2] : "",
 			year: date.length === 3 ? date[0] : "",
+			focused: date.length === 3,
 			monthArr: Array.from({ length: 12 }, (v, k) => k + 1),
-			dayArr: Array.from({ length: 31 }, (v, k) => k + 1),
-			yearArr: Array.from({ length: 101 }, (v, k) => new Date().getFullYear() - k),
-			focused: date.length === 3
+			yearArr: Array.from({ length: 20 }, (v, k) => new Date().getFullYear() + k) // We only need about 20 years ahead... right?
 		};
 	}
 
-		static propTypes = {
-			value: PropTypes.object,
-			placeholder: PropTypes.string.isRequired,
-			onSelect: PropTypes.func.isRequired
-		}
+	static propTypes = {
+		value: PropTypes.object,
+		placeholder: PropTypes.string.isRequired,
+		onSelect: PropTypes.func.isRequired
+	}
 
-		prepend0(item) {
-			if (item < 10)
-				return "0" + item;
+	// Returns an array of days in the current state's month when called
+	dayArr() {
```

Comment:
This name should describe what the function is doing. It should be something like `getArrayOfDays`